docs/src/test/scala/docs/http/scaladsl/HttpClientDecodingExampleSpec.scala

```diff
@@ -47,6 +47,8 @@ class HttpClientDecodingExampleSpec extends WordSpec with Matchers with CompileO
 
     val futureResponses: Future[Seq[HttpResponse]] =
       Future.traverse(requests)(http.singleRequest(_).map(decodeResponse))
+
+    system.terminate()
```

Comment:
I didn't look much on the Scala example as it was already there and approved. I agree with you, maybe it makes sense to collect the answers and log them.